views/no-strike-off.html

```diff
@@ -0,0 +1,32 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  No strike off notice
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        There is no strike off notice for the company
+      </h1>
+      <p class="govuk-body-l">Check the status of {{ companyName }}
+        again in approximately 4 weeks' time. If dissolution action has started, you can use this service to apply
```

Comment:
weeks' - the apostrophe looks wrong, looks like weekses.

Reply:
I agree, but it's in the screenshot. Leave for now I say but will raise this with Sarah and Karen.

Reply:
Checked with Sarah and also consulted Google. It's needed - https://www.sarahtownsendeditorial.co.uk/2015/11/using-an-apostrophe-time-expressions/.